tests/tensorflow2/test_keras.py

```diff
@@ -155,6 +155,9 @@ def helper_keras_gradtape(
 
     opt = hook.wrap_optimizer(opt)
 
+    tape = tf.GradientTape(persistent=True)
+    tape = hook.wrap_tape(tape)
+
```

Comment:
See comment above about making this a one-liner.

Reply:
done in the latest commit

Reply:
Is persistent=True still necessary?

Reply:
not necessary (by default persistent=False). changing it in the latest commit.

added a test for persistent=True and made a few changes to handle multiple calls to tape.gradient in this case.